### ISSUE TITLE
Spine stiffness

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6542]
**Preferred term label:** Spine stiffness

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Joint stiffness HP:0001387

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
There is a variability in the way the word sitffness is used. Here is an example of the way we have defined it in the HPO:
A sensation of tightness in the ankle joint when attempting to move it, especially after a period of inactivity.
https://hpo.jax.org/app/browse/term/HP:0025264
would an analogous definition be appropriate here?
I have looked for a PMID, but many of the articles describe mechanical measurements that are not exactly the same definition. Possibly it would be clearer to say reduced range of motion?

**Comment by pnrobinson** [comment]
added term
